firmware/application/apps/ui_fileman.hpp

```diff
@@ -107,17 +110,23 @@ class FileManBaseView : public View {
     bool show_hidden_files{false};
 
     Labels labels{
-        {{0, 0}, "Path:", Color::light_grey()}};
+        {{0, 0}, "@", Color::light_grey()}};
 
     Text text_current{
-        {6 * 8, 0 * 8, 24 * 8, 16},
+        {1 * 8, 0 * 8, 20 * 8, 16},
         "",
     };
 
     MenuView menu_view{
         {0, 2 * 8, 240, 26 * 8},
         true};
 
+    OptionsField option_profile_switch{
```

Comment:
The GUI position is not settled. I will change it after all implemented.
Layout suggestions is really welcomed.